tgui/packages/tgui/interfaces/Uplink/PrimaryObjectiveMenu.tsx

```diff
@@ -0,0 +1,60 @@
+import { Component, Fragment } from 'inferno';
+import { Box, Flex, NoticeBox, Section, Stack } from '../../components';
+
+type PrimaryObjectiveMenuProps = {
+  primary_objectives,
+  final_objective,
+};
+
+export class PrimaryObjectiveMenu extends Component<
+  PrimaryObjectiveMenuProps
+> {
+  render() {
+    const {
+      primary_objectives,
+      final_objective,
+    } = this.props;
+    return (
+      <Fragment>
+        {!!final_objective &&(
+          <Fragment>
+            <Box
+              width={"100%"}
+              hight={"100%"}
+              position="absolute"
+              className="UplinkObjective__EmptyObjective" />
+            <NoticeBox
+              position="absolute"
+              width={"100%"}
+              fontSize="30px"
+              textAlign="center">
+              ALL OBJECTIVES MARKED COMPLETE 
+            </NoticeBox>
+          </Fragment>
```

Comment:
this notice box is really ugly, also I don't think "ALL OBJECTIVES MARKED COMPLETE" is a good phrase to use.
It may just be preferable to have nothing here, if not just displaying the final objective instead.